export/README.md

```diff
@@ -0,0 +1,51 @@
+# MuJoCo Export Pipeline
+This code generates a MuJoCo replica of a SPEAR scene.
+
+## Usage
+All commands to be run from the `export` directory. Assume that the name of the scene is stored in the `SCENE_NAME`
+environment variable.
+- If you have not done so already, follow the SPEAR [Getting Started](../docs/getting_started.md) instructions, which
+will result in the `spear-env` conda environment.
+- Build V-HACD using the [readme instructions](v-hacd/README.md).
```

Comment:
If V-HACD is required to run this code, it should be added to the `third_party` folder and should be built automatically by `build_third_party_libs.py`